lib/config.js

```diff
@@ -7,7 +7,10 @@ var config = require('bedrock').config;
 var path = require('path');
 require('bedrock-validation');
 
-config['key-http'] = {};
+// var basePath = '/keys';
+// config['key-http'] = {};
+// config['key-http'].routes = {};
+// config['key-http'].routes.basePath = basePath;
```

Comment:
I might need Matt's input on this, but in general, I think the path (called basePath here, but it should be renamed keyPath) does need to be made into a computed config variable.  I'll talk to Matt about it next week and resolve this.  I'll either update, or make a note in a comment on why that commented code is there.